src/OAuthLibrary/OAuthControl/OAuthWebBrowser.cs

```diff
@@ -40,6 +41,26 @@ public class OAuthWebBrowser : WebBrowser
         /// </summary>
         public string ClientId { get; set; }
 
+        /// <summary>
+        /// The Org to login to
+        /// </summary>
+        public string Org { get; set; }
```

Comment:
These new properties should be documented in the readme. The existing ones are under the Setup heading.